wiki/osu!_wiki/Contribution_guide/Best_practices/en.md

```diff
@@ -30,7 +30,7 @@ If you encountered any problems while using the GitHub tool or you want to overw
 2. In `Workflows`, look for `Sync from osu! upstream`.
 3. Click `Run workflow` and fill in the options:
 
-![](img/github-actions-workflow-dialog.png "GitHub Actions Workflow - Run Workflow")
+![](img/github-actions-workflow-dialog.png "GitHub Actions Workflow—Run Workflow")
```

Comment:
Is this a similar case to the image title mentioned elsewhere? Should the em dash also be reverted here?

Reply:
yeah, revert this

Reply:
```suggestion
![](img/github-actions-workflow-dialog.png "GitHub Actions Workflow - Run Workflow")
```